storage/mysql/log_storage_test.go

```diff
@@ -440,11 +441,40 @@ func TestAddSequencedLeavesWithDuplicates(t *testing.T) {
 	aslt.verifySequencedLeaves(6, 4, dupLeaves)
 }
 
+func BenchmarkAddSequencedLeaf(b *testing.B) {
```

Comment:
I'm not sure we'll see differences on a small database but it's worth a try and does exercise the code. With some of my test runs performance impacts only became visible at about 300,00 - 500,000 leaves.